### PR TITLE
Document data manager repository layout

### PR DESCRIPTION
**Pull request by innovate-invent**

### CODE REVIEW

File: doc/source/dev/data_managers.rst [hunk 1]
```diff
@@ -530,6 +536,15 @@ referenced by the Data Manager Tool, if any, is stored within the
 A data manager tool can use a ``conda`` environment if the target
 Galaxy is version 18.09 or above (specified in the tool's XML file).
 
+Data manager tools are loaded by referring to them in the data_manager_conf.xml file for the repository.
+
+Data table configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Data tables that the data managers operate on are specified in files at `tool-data/*.loc.sample` relative to the repository root.
```
Comment (mvdbeek) [inline doc/source/dev/data_managers.rst]: The loc files are samples of how tabular data tables should look like.
The `tool_data_table_conf.xml.sample` at the repository root defines the data tables.

Comment (innovate-invent) [reply doc/source/dev/data_managers.rst]: Is that search using the same function as the data_manager_conf.xml?
Why is tool_data_table_conf.xml.sample suffixed with .sample but data_manager_conf.xml isnt?

Comment (mvdbeek) [reply doc/source/dev/data_managers.rst]: `data_manager_conf.xml` describes the data manager you've written. `tool_data_table_conf.xml.sample` is copied to shed_tool_data_path and filled with details about the new tool data table.

Comment (innovate-invent) [reply doc/source/dev/data_managers.rst]: So it is the data_manager_conf.xml that specifies the data table? And tool_data_table_conf.xml.sample is strictly a sample table config that could possibly pre-populate data tables?

Is tool_data_table_conf.xml.sample optional?

Comment (mvdbeek) [reply doc/source/dev/data_managers.rst]: `data_manager_conf.xml` references the tool that generates data and how to move generated data and fill the loc file programmatically.
If you want to pre-populate a data table you'd add entries to the loc file sample.

**Comment by mvdbeek** [comment]
Awesome, thank you!

**Pull request merged**